Cargo.toml

```diff
@@ -15,6 +15,9 @@ opt-level = 3
 panic = 'abort'
 incremental = false
 
+[patch.crates-io]
+iroh-io = { path = "./iroh-io" }
```

Comment:
Without this the dependency iroh-io of bao tree and the local stuff does not get unified. Not sure what to do about that.

Reply:
actually it is pretty clear - have a dependency on crates.io everywhere.